vmm/src/config.rs

```diff
@@ -2339,12 +2339,12 @@ impl VmConfig {
 
         let mut net: Option<Vec<NetConfig>> = None;
         if let Some(net_list) = &vm_params.net {
-            let mut net_config_list = Vec::new();
+            let mut net_fixture_list = Vec::new();
```

Comment:
I would rather you don't change this bit - there is a common convention in this code.

Reply:
Find and replace error. :)